developer-docs-site/docs/tutorials/run-a-fullnode-on-gcp.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Run a FullNode on GCP"
+slug: "run-a-fullnode-on-gcp"
+sidebar_position: 11
+---
+
+# Run a FullNode on GCP
+
+This tutorial explains how to configure and deploy a public FullNode to connect to the Aptos devnet using Google Cloud (GCP). Running Fullnode on Cloud usually provides better stability and avaliability compare to running it on your laptop, if you're looking for deploying a production grade Fullnode, we recommend you to deploy it on Cloud.
+
+> **Note:** Please read [Run a Fullnode](run-a-fullnode) if you want other alternatives for deployment, using Cloud comes with a cost, and it varies depends on how you configure it.
+>
+
+## Prerequisites
+Before you get started with this tutorial, install the required dependencies and get familiar with the toolings:
+* Terraform 1.1.7: https://www.terraform.io/downloads.html
+* Kubernetes cli: https://kubernetes.io/docs/tasks/tools/
+* Google Cloud cli: https://cloud.google.com/sdk/docs/install-sdk
+
+Once you have installed the gcloud CLI, log into GCP using gcloud (https://cloud.google.com/sdk/gcloud/reference/auth/login)
+```
+$ gcloud auth login --update-adc
+```
+
+## Getting started
+
+You can deploy a public FullNode on GCP by using the Aptos fullnode Terraform module.
+
+1. Create a working directory for your configuration.
+
+    * Choose a workspace name e.g. `devnet`. Note: this defines terraform workspace name, which in turn is used to form resource names.
+    ```
+    $ export WORKSPACE=devnet
+    ```
+
+    * Create a directory for the workspace
+    ```
+    $ mkdir -p ~/$WORKSPACE
+    ```
+
+2. Create a storage bucket for storing the Terraform state on Google Cloud Storage.  Use the console or this gcs command to create the bucket.  See the Google Cloud Storage documentation here: https://cloud.google.com/storage/docs/creating-buckets#prereq-cli
+  ```
+  $ gsutil mb gs://BUCKET_NAME
+  ```
+
+3. Create Terraform file called `main.tf` in your working directory:
+  ```
+  $ cd ~/$WORKSPACE
+  $ touch main.tf
+  ```
+
+4. Modify `main.tf` file to configure Terraform, and create fullnode from Terraform module. Example content for `main.tf`:
+  ```
+  terraform {
+    required_version = "~> 1.1.0"
+    backend "gcs" {
+      bucket = "aptos-terraform-dev" # bucket name created in step 2
```

Comment:
replace with `BUCKET_NAME` to make it more obvious that they need to replace it?